core/modules/field/field.module

```diff
@@ -622,24 +623,38 @@ function field_sync_field_status() {
  *
  * @param $module
  *   The name of the module to update on.
+ * @param $fields
+ *   An optional array of field definitions to operate on. If left empty, all
+ *   fields will be loaded.
  */
```

Comment:
```suggestion
 * @param $fields
 *   An optional array of field definitions to operate on. If left empty, all
 *   fields will be loaded.
 *
 * @since 1.18.4 Second parameter $fields added for performance optimization.
 */
```